mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/44 - main UI

Closes #44 

To test:
- Run Starting Scenarios/Introduction Scenario scenario from Main Menu
- Make sure pause button pauses/unpauses simulation (should start paused once the server is connected)
- Funds and bonus per hour should be updated properly once simulation is unpaused
- Time label (upper left) should be updated with the information we get from the server

Notes:
- All the buttons on the outer UI fire events, but nothing is listening to those events until those UI elements get created
- Current scenario/current campaign, and current time/building information is not being sent from server, so the matching UI can't update properly until that is fixed
